lib/scimgateway.js

```diff
@@ -788,7 +788,7 @@ const ScimGateway = function () {
           logger.debug(`${gwName}[${pluginName}] calling "${handler.groups.getMethod}" and awaiting result - groups to be included`)
           let res
           try {
-            res = await this[handler.groups.getMethod](ctx.params.baseEntity, { attribute: 'members.value', operator: 'eq', value: getObj.value }, ['members.value', 'id', 'displayName'])
```

Comment:
Will this impact the `createGroup` or `modifyGroup` implementation in our SCIM Gateway?

Reply:
I've had to make changes to our overlay on the SCIM Gateway to deal with these changes. Primarily the getGroups function. You can review the internal PR to see those changes